### ISSUE TITLE
Float Input to 

### ISSUE THREAD

**Issue by trustinbharat** [fchollet/keras#2562]
The following is the example in the documentation for comparison of tweets. I wish to use similar set up for feature vector comparison. There are two feature vector has 100 attributes describing facial features of a person like Asian, Indian etc. The quantities are in float numbers.
The goal is to identify if two feature vectors are of same person or not.

However, in the example below, the input is discrete in 256 dimensions.

Please tell me how to remodel the same for my problem.

Many Thanks

from keras.layers import Input, LSTM, Dense, merge
from keras.models import Model

tweet_a = Input(shape=(140, 256))
tweet_b = Input(shape=(140, 256))
shared_lstm = LSTM(64)

encoded_a = shared_lstm(tweet_a)
encoded_b = shared_lstm(tweet_b)

merged_vector = merge([encoded_a, encoded_b], mode='concat', concat_axis=-1)

predictions = Dense(1, activation='sigmoid')(merged_vector)

model = Model(input=[tweet_a, tweet_b], output=predictions)

model.compile(optimizer='rmsprop',
              loss='binary_crossentropy',
              metrics=['accuracy'])
model.fit([data_a, data_b], labels, nb_epoch=10)

**Comment by rpinsler** [comment]
It's basically the same, except that you want to do regression, see #2399 for an example.

**Comment by trustinbharat** [comment]
Thanks rpinsler for quick response.
I didn't understand the link you have sent. I will explain it again. I have set of instances <feature vector 1, feature vector 2>. Each feature vector has 100 attributes with floating values. The labels are <0,1> describing both feature vector belong to same person or not. In the example I quoted, it shows the tweet, feature attribute is one out of 256 values. In my case each attribute can have floating number. I need to know how should I be training my NN for this

**Comment by rpinsler** [comment]
Yeah, I realized it wasn't helpful, which is why I have deleted it in the meantime :) I think the above example is very similar to what you want to do. Since you don't have time-dependencies your input shape is simply Input(shape=(100,)). Furthermore, you have to replace the LSTM, e.g. by Dense layers.

**Comment by trustinbharat** [comment]
Hi,

I tried that as suggested, but I am getting this error
when running 
model.fit([data_a, data_b], labels, nb_epoch=10)
Input dimension mis-match. (input[0].shape[1] = 2, input[1].shape[1] = 1)
